assets/blocks/toggle-legacy-course-metaboxes-wrapper.js

```diff
@@ -13,7 +16,7 @@ const ToggleLegacyCourseMetaboxesWrapper = ( {
 	children,
 } ) => {
 	useEffect( () => {
-		if ( isPreview ) {
+		if ( isPreview || ! window.sensei_toggle_legacy_metaboxes ) {
```

Comment:
I didn't like this solution because the Contact Teacher button should not have the `EditWrapper: ToggleLegacyCourseMetaboxesWrapper` when it's in the course. Since this button is shared in more than one post type, I think this logic should be outside of it, but I thought/tried a lot of different approaches and I didn't find any good solution. The main reasons are:

- We need to register the block before we know what's the custom post type of the page. One solution would be creating a global variable through another script imported only in the course (or an inline script), but it also seems a very ugly solution.
- We don't want to have different buttons for both situations because sometimes we'll remove this toggle and both need to continue being the same button.